### DISCUSSION TITLE
Wrong header in drop down while browsing in mobile

### DISCUSSION THREAD

**Discussion by omoju** [fimio-xyz/feedback#6] [Bug]
When I am browsing the fimio.xyz site on my mobile device, when I click on the drop down menu, I see `argon` and an unfamiliar logo. I think this is a bug.
![IMG_1123](https://user-images.githubusercontent.com/1815882/231593412-c2c5eaf0-a0f0-4a4d-8a52-a62ad8a96dc4.png)

**Comment by joshuaiokua** [comment]
Thanks for bringing this bug to our attention! It has been addressed as shown below.
![IMG_4C3A2B0ADCBF-1](https://user-images.githubusercontent.com/32827030/231595342-2a37f2f6-9dde-4d4b-ad39-527df03d08bc.jpeg)
